menu.js

```diff
@@ -0,0 +1,38 @@
+const lastWidth = document.querySelector('body').style.width;
+
+const closeMenu = function () {
+  document.getElementsByClassName('menu')[0].classList.add('hidden');
+  document.querySelector('body').style.width = lastWidth;
+  document.getElementsByClassName('headline')[0].classList.remove('hidden');
+  document.getElementsByClassName('Works')[0].classList.remove('hidden');
+  document.getElementsByClassName('About')[0].classList.remove('hidden');
+  document.getElementsByClassName('form')[0].classList.remove('hidden');
+};
+
+const openMenu = function () {
+  document.getElementsByClassName('menu')[0].classList.remove('hidden');
+  document.querySelector('body').style.width = '10vmax';
+  document.getElementsByClassName('headline')[0].classList.add('hidden');
+  document.getElementsByClassName('Works')[0].classList.add('hidden');
+  document.getElementsByClassName('About')[0].classList.add('hidden');
+  document.getElementsByClassName('form')[0].classList.add('hidden');
+};
+
+document
+  .getElementsByClassName('cancel')[0]
+  .addEventListener('click', closeMenu);
+document
+  .getElementsByClassName('menu-button-selector')[0]
+  .addEventListener('click', openMenu);
```

Comment:
- [ ] As suggested by the previous reviewer. Please ensure that your mobile menu responsive until it changes to desktop screen size. This means the Close Menu Button should remain at the top right corner when increasing the device width.